scripts/jenkins/ardana/ansible/roles/ardana_update/tasks/reboot_nodes.yml

```diff
@@ -0,0 +1,58 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Reboot deployer
```

Comment:
Would be nice to have this defined in a separate role, to be able to do just a node reboot.

Reply:
Yes, I agree. We can do it later when there is request for it :)